spec/features/add_space_spec.rb

```diff
@@ -1,25 +1,23 @@
-feature 'Add space' do
-  let(:name){'MY SPACE'}
-  let(:description){'description'}
-  let(:price){'12.11'}
-  let(:date){'2016-05-01'}
+feature 'Add space form:' do
+  let(:name){'first house'}
+  let(:description){'pretty'}
+  let(:price){'10.01'}
+  let(:date){'11 may 2016'}
 
   before :each do
-    named_signup
-    visit '/spaces/new'
+    visit '/'
+    click_button('Register')
+    submit_signup_form("host","host","h@m.com","host_pass","host_pass")
   end
 
-  scenario 'to table' do
-    fill_in :space_name, with: name
-    fill_in :space_description, with: description
-    fill_in :space_price, with: price
-    fill_in :space_availability, with: date
-    click_button 'Submit'
+  scenario 'new space gets listed on the main page' do
```

Comment:
Maybe let's add a start page at the beginning of each scenario in case we end up changing where a user is redirected after sign up, i.e., begin the scenario with ```visit '/spaces'```.